packages/driver/cypress/integration/e2e/multi-domain/multi_domain_spec.ts

```diff
@@ -327,10 +265,5 @@ describe('multi-domain', { experimentalSessionSupport: true, experimentalMultiDo
         })
       })
     })
-
-    // TODO: this following tests needs to be implemented in a cy-in-cy test or more e2e style test as we need to test the 'done' function
```

Comment:
Probably would be a good idea to add cy-in-cy tests once we're merged with the 10.0 tests. I removed these tests since they're related to done callbacks.